android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java

```diff
@@ -78,30 +76,22 @@ public void open(Context context, final ReadableMap options, final Promise promi
       return;
     }
 
-    CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
+    CustomTabColorSchemeParams.Builder colorSchemeBuilder = new CustomTabColorSchemeParams.Builder();
     if (options.hasKey(KEY_TOOLBAR_COLOR)) {
-      final String colorString = options.getString(KEY_TOOLBAR_COLOR);
-      try {
-        builder.setToolbarColor(Color.parseColor(colorString));
-        isLightTheme = toolbarIsLight(colorString);
-      } catch (IllegalArgumentException e) {
-        throw new JSApplicationIllegalArgumentException(
-                "Invalid toolbar color '" + colorString + "': " + e.getMessage());
-      }
+      final int color = options.getInt(KEY_TOOLBAR_COLOR);
+      colorSchemeBuilder.setToolbarColor(color);
```

Comment:
how are you handling exceptions here?

Reply:
Now that colors are processed in JS invalid values are transformed to undefined so it never gets here.

Reply:
The main difference is that the color will be ignore instead of throwing, but this is the behavior for iOS colors already so I think this is correct.

Reply:
I prefer to add missing validations for iOS colors instead of removing these validations for Android :)

Reply:
I'll check if react-native has a method to validate it. I guess it could also be possible to check if the input of processColor is not undefined and the result is.

Reply:
I added a warning in dev instead of an error, I think this is better. Let me know what you think.

<img width="373" alt="image" src="https://user-images.githubusercontent.com/2677334/96273939-ecad6200-0f9d-11eb-9fc9-f13379657d6b.png">

Reply:
If we don't have a `react-native` method to validate the color, I prefer to have these validations from native code :)